modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -0,0 +1,246 @@
+{{#platforms}}
+{{#node}}
+// TODO: evaluate if we can easily get rid of this library
+import * as FormData from "form-data";
+{{/node}}
+{{/platforms}}
+// typings of url-parse are incorrect...
+// @ts-ignore 
+import * as URLParse from "url-parse";
```

Comment:
is this really necessary in the browser?

Reply:
In most browsers we probably could use URL.searchParams, except for IE.
This way we can use the same for all environments.
Not sure if there is a URL.searchParams polyfill for both IE and node.

Reply:
There's `https://www.npmjs.com/package/url-search-params-polyfill` which works for both IE and node, has no dependencies, and actually checks if `URL.searchParams` is available before using its own code. 

Alternatively, we could also just expect people to provide their own polyfill for IE, i.e. we only use URLParse in node and assume that URLParse is available in the browser

Which option would you prefer?

Reply:
Edit: It would be a giant pain to re-implement this without a polyfill - Given that `url-parse` has exactly 2 dependencies (including recursive dependencies), I'd propose to just use url-parse.

Reply:
I think it is fine for now, we can always improve this later, by using `url-serach-params-polyfill` instead and maybe add a configuration option to enable polyfills or not.